src/analyzer/engines/sklearn.py

```diff
@@ -0,0 +1,80 @@
+from src.analyzer.engines.engine import ModelingEngine
+
+from sklearn.model_selection import GridSearchCV
+from sklearn.decomposition import LatentDirichletAllocation
+from sklearn.decomposition import NMF
+
+class SklearnEngine(ModelingEngine):
+    """
+    GensimEngine implements the methods for fitting and evaluating gensim models on a
+    corpus.
+    """
+    def __init__(self, min_topics, max_topics):
+        self.min_topics = min_topics
+        self.max_topics = max_topics
+        self.grid = None
+        self.model = None
+        self.vectorizer = None
```

Comment:
Suggest making these "learned attributes" in the sklearn style (trailing underscore)